tests/refarm_pagination/test_views.py

```diff
@@ -0,0 +1,65 @@
+from contextlib import contextmanager
+from unittest import mock
+
+from django.conf import settings
+from django.test import TestCase
+from refarm_pagination import views, pagination
+
+from tests.catalog.models import MockProduct
+
+
+class PaginationContext(TestCase):
+
+    fixtures = ['catalog.json']
+
+    def context(self, **kwargs):
+        return views.PaginationContext(**{
+            'url': '',
+            'number': 1,
+            'per_page': 1,
+            'objects': MockProduct.objects.all(),
+            **kwargs
+        }).context()
+
+    @contextmanager
+    def mock_neighbor_pairs(self, url, number):
+        with \
```

Comment:
we don't have a convention about it, but in such cases we always used brackets instead of slashes

Reply:
We have three options to implement this:

```python
In [27]: with (
    ...:     p('__main__.open')) as b, (
    ...:     p('__main__.open')) as c:
    ...:     pass
    ...:
    ...:

In [28]: with \
    ...:     p('__main__.open') as b,\
    ...:     p('__main__.open') as c:
    ...:     pass
    ...:
    ...:

In [29]: with p('__main__.open') as b:
    ...:     with p('__main__.open') as c:
    ...:         pass

In [30]: with (p('__main__.open') as b,
    ...:     p('__main__.open') as c):
    ...:     pass
    ...:
    ...:
  File "<ipython-input-29-ec9ee7894e84>", line 2
    p('__main__.open') as b,
                        ^
SyntaxError: invalid syntax
```

It seems the second one is better